hub/jupyterhub_config.py

```diff
@@ -13,6 +13,28 @@
 # Our simplest user image! Optimized to just... start, and be small!
 c.KubeSpawner.singleuser_image_spec = 'yuvipanda/simple-singleuser:v1'
 
+# Configure dynamically provisioning pvc
+c.KubeSpawner.pvc_name_template = 'claim-{username}-{userid}'
+c.KubeSpawner.storage_class = 'single-user-storage'
+c.KubeSpawner.access_modes = ['ReadWriteOnce']
+c.KubeSpawner.storage = '10GiB'
+
+# Add volumes to singleuser pods
+c.KubeSpawner.volumes = [
+    {
+        'name': 'volume-{username}-{userid}',
+        'persistentVolumeClaim': {
+            'claimName': 'claim-{username}-{userid}'
+        }
+    }
+]
+c.KubeSpawner.volume_mounts = [
```

Comment:
Specify the directory of the notebook container pod that the PVC should be mounted on. (in this case ```/home```)